### PR TITLE
Docs/flight methodology

### PR DESCRIPTION
**Pull request by Mike3563**
### What does this PR do ?
Document flight methodology

### CODE REVIEW

File: docs/flight-methology.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+id: flight-methology
```
Comment (xavier506) [inline docs/flight-methology.md:2]: ---
id: flight-methodology
title: Flight Methodology
sidebar_label: Flight Methodology
---

**Comment by xavier506** [comment]
@Mike3563  the name of the file is incorrect `docs/flight-methology.md`

File: docs/flight-methology.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+title: Flight Metholog
+sidebar_label: Flight Metholog
+---
+
+## Edenia agile metology!
+
+
+We at Edenia are a company that uses agile methodologies that can be adapted to our flexible way of working.
+Since we are not rigid when it comes to change and uncertainty, so we use a hybrid of different agile methodologies applied and tropicalized to our needs in processes.
```
Comment (xavier506) [inline docs/flight-methology.md:11]: Maybe we can include something like this

we also like to be innovative not just with our software solutions but with our processes , constantly evolving and improving the way our team is organized to optimize how we deliver results.

File: docs/flight-methology.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+- **Staff:** This member can be anyone on the team working on a task on the flight.
+
+## Some recomendation about flights methodoly are:
+
+1. **Every flight must have a pilot**
+2. **A developer can only be a pilot of one flight at a time (but can be a co-pilot or staff)**
+3. **Developers can be staff on different flights at the same time**
+
+## Flyght status
```
Comment (xavier506) [inline docs/flight-methology.md:57]: Flight

File: docs/flight-methology.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+3. **Developers can be staff on different flights at the same time**
+
+## Flyght status 
+
+- **On Deck:** When we have an idea for a project we integrate a new flight to the document.
+- **At Gate:** Clear idea about the flight, what must be done to add value to the project.
+- **Boarding:** Indicates that as soon as someone finishes a flight, it will be a high priority to make the new flight.
+- **Flying:** When you are already working on the flight and you have a pilot, co-pilot and staff.
+- **Delayed:** the flight has external or internal complications, it will not be able to land on time.
+- **Landed:** this means the flight task was completed and the result was satisfactory
```
Comment (xavier506) [inline docs/flight-methology.md:64]: flight objective rather than task (flight has reached its destination)

File: docs/flight-methology.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+As a general rule, it makes sense to book a flight with tasks that go together, depend on each other, or are expected to land at the same time.
+However, nothing prevents you from adding unrelated tasks to a flight if there is "room for it".
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/pCShsesxRZA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Flight Crew
+
+- **Pilot:** It can be any person from the team, this person is in charge of delivering the flight with all the requirements and on time.
+
+- **Co- Pilot:** As co-pilot, he must give support to the pilot, if necessary, and between the two of them, look for solutions if they encounter any turbulence.
```
Comment (xavier506) [inline docs/flight-methology.md:47]: Co-Pilot (remove space)

**Comment by xavier506** [comment]
@Mike3563  we are also going to need to link to this page from another page in order for people to find it , it can also be included as a sidebar menu item

File: docs/flight-methology.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+## Some recomendation about flights methodoly are:
+
+1. **Every flight must have a pilot**
+2. **A developer can only be a pilot of one flight at a time (but can be a co-pilot or staff)**
+3. **Developers can be staff on different flights at the same time**
+
+## Flyght status 
+
+- **On Deck:** When we have an idea for a project we integrate a new flight to the document.
+- **At Gate:** Clear idea about the flight, what must be done to add value to the project.
```
[the remark this answers is not in the document]
Comment (Mike3563) [reply docs/flight-methology.md:60]: Code review change

File: docs/flight-methology.md (hunk 4 above)
Comment (Mike3563) [reply docs/flight-methology.md:64]: Code review change

**Review by Mike3563 — COMMENTED** [review]
Code review change

File: docs/flight-methodology.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
+---
+id: flight-methology
```
Comment (xavier506) [inline docs/flight-methodology.md:2]: methology is misspelled

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**